openshift/templates/icbc-web-adapter.dc.yaml

```diff
@@ -0,0 +1,157 @@
+  - name: MEMORY_LIMIT
+    displayName: Memory Limit
+    description: Maximum amount of memory the container can use.
+    required: true
+    value: 128Mi
+  - name: NAMESPACE
+    displayName: Namespace
+    description: The OpenShift Namespace where the ImageStream resides.
+    required: true
+    value: openshift
+  - name: APPLICATION_DOMAIN
+    displayName: Application Hostname
+    description:
+      The exposed hostname that will route to the .NET Core service, if left
+      blank a value will be defaulted.
+    value: ""
+  - name: TAG
+    displayName: Image Tag
+    description:
+      The image tag to be deployed from the registry.
+    required: true
+    value: latest
+  - name: JAEGER_COLLECTOR
+    displayName: Jaeger Collector Service
+    description:
+      The name of the Jaeger collector to send traces to (port 14268)
+    required: true
+    value: jaeger-collector
+objects:
+  - kind: Route
+    apiVersion: v1
+    metadata:
+      name: "${NAME}"
+    spec:
+      host: "${NAME}-${APPLICATION_DOMAIN}"
+      to:
+        kind: Service
+        name: "${NAME}"
+    port:
+      targetPort: web
+    tls:
+      insecureEdgeTerminationPolicy: Redirect
+      termination: edge
+  - kind: Service
+    apiVersion: v1
+    metadata:
+      name: "${NAME}"
+      annotations:
+        description: Exposes and load balances the application pods
+    spec:
+      ports:
+        - name: web
+          port: 8080
+          targetPort: 8080
+      selector:
+        name: "${NAME}"
+  - kind: DeploymentConfig
+    apiVersion: v1
+    metadata:
+      name: "${NAME}"
+      annotations:
+        description: Defines how to deploy the application server
+    spec:
+      strategy:
+        type: Rolling
+      triggers:
+        - type: ConfigChange
+        - type: ImageChange
+          imageChangeParams:
+            automatic: true
+            containerNames:
+              - dotnet-runtime-app
+            from:
+              kind: ImageStreamTag
+              namespace: "${NAMESPACE}"
+              name: "${NAME}:${TAG}"
+      replicas: 1
+      selector:
+        name: "${NAME}"
+      template:
+        metadata:
+          name: "${NAME}"
+          labels:
+            name: "${NAME}"
+        spec:
+          containers:
+            - name: dotnet-runtime-app
+              image: " "
+              ports:
+                - containerPort: 8080
+              resources:
+                limits:
+                  memory: "${MEMORY_LIMIT}"
+              env: 
+                - name: JAEGER_SERVICE_NAME
+                  value: "${NAME}"
+                - name: JAEGER_SAMPLER_TYPE
+                  value: const
+                - name: JAEGER_ENDPOINT
+                  value: http://${JAEGER_COLLECTOR}:14268/api/traces
+                - name: ASPNETCORE_ENVIRONMENT
+                  value: Development
+                - name: RABBITMQ__HOST
+                  value: rabbitmq-cluster
```

Comment:
I see a diff here with the other ones that are using RABBIT_MQ, I guess not much impact